src/view/document.js

```diff
@@ -74,16 +74,27 @@ export default class Document {
 		this.roots = new Map();
 
 		/**
-		 * {@link engine.view.EditableElement} which is currently focused or null if all of them are blurred.
+		 * True if document is focused.
 		 *
 		 * This property is updated by the {@link engine.view.obsever.FocusObserver}.
 		 * If the {@link engine.view.obsever.FocusObserver} is disabled this property will not change.
 		 *
 		 * @readonly
 		 * @observable
-		 * @member {engine.view.EditableElement|null} engine.view.Document#focusedEditable
+		 * @member {Boolean} engine.view.Document#isFocused
 		 */
-		this.set( 'focusedEditable', null );
+		this.set( 'isFocused', false );
+
+		/**
+		 * {@link engine.view.EditableElement EditableElement} which is containing selection. It can be `null` if
+		 * there is no selection found.
+		 *
+		 * @readonly
+		 * @observable
+		 * @member {engine.view.EditableElement|null} engine.view.Document#selectedEditable
+		 */
+		this.set( 'selectedEditable', null );
```

Comment:
Why `null` by default? This means that the `selectedEditable` will be `null` as long selection does not change. It should be `this.selection.getEditableElement()` by default.